pages/summary-confirmation.js

```diff
@@ -12,6 +12,42 @@ import PropTypes from "prop-types";
 import styled from "react-emotion";
 import moment from "moment";
 
+const wales = [
```

Comment:
We recently moved such configuration data to the back-end config db to avoid hard-coding in the application itself. I would suggest doing something similar here. A new "country" field should be added to the existing "lcConfig" collection, which stores configuration for each council. A new function would need to be created in the config-db.connector to retrieve the "country" field given the council's name. That function should then be invoked in the new.route and saved in the session. Since we currently manage the Azure resources, we would be happy to make the necessary changes there. It's up to you if you want to implement the rest of this change or if we should do that too.